src/test/java/uk/gov/register/presentation/functional/RepresentationsTest.java

```diff
@@ -50,7 +51,7 @@ public RepresentationsTest(String extension, String expectedContentType, String
         this.expectedListOfEntries = expectedListOfEntries;
     }
 
-    @Test
+    @Ignore
```

Comment:
I didn't want to do the hard work again very soon when we write other representations for new resources. When we play the representations card this test will be unignored.